src/js/profile/facilities/facility_controller.js

```diff
@@ -84,11 +84,13 @@ export class FacilityController extends Component {
     }
 
     getAndAddFacilities() {
-        this.model.api.getThemesCount(this.model.profileId, this.model.geography.code)
-            .then((data) => {
-                this.model.themes = data;
-                this.addFacilities();
-            })
+        if(this.model.api !== null){
```

Comment:
when would this be null?

Reply:
jest throws exception because it does not recognize `fetch` and I didn't want to import `jest-fetch-mock` just for this so I just send `null` so the api is not called - what do you think about this workaround?
https://github.com/OpenUpSA/wazimap-ng-ui/blob/741ae253e7b82a98a8ddf8382c402a59ca640e68/__tests__/profile/profile_header.test.js#L129